applications/Makefile

```diff
@@ -22,13 +22,13 @@ CLEAN_APPS= $(APPS:%=clean-%)
 
 EXTRA= 	12swap ntetris pipepanic \
 	madplay sprsht flchekers \
-	yareversi opendune linapple \
-	dosbox dillo \
-	martian jooleem.sh tuxchess cr3 \
-	sudoku checkers freecell nxpad_gnuboy \
-	eggchess.sh navit net-bubble
+	yareversi linapple \
+	martian jooleem.sh tuxchess \
+	sudoku checkers freecell  \
+	cr3  dillo eggchess.sh navit net-bubble
 
-BROKEN_EXTRA= gnuboy scummvm
+DISABLED_EXTRA= dosbox dillo nxpad_gnuboy
```

Comment:
dillo was a mistake. I added it at EXTRA too and forgot here.
dosbox download source was broken but work now again.
nxpad_gnuboy compiled every run again.